tests/model/test_time_to_triage.py

```diff
@@ -0,0 +1,96 @@
+import copy
+import datetime
+
+from treeherder.perf.models import PerformanceAlertSummary
+
+
+class TestTimeToClass:
+    def test_alert_summary_created_monday(self, test_perf_alert_summary):
+        test_perf_alert_summary.created = datetime.datetime.fromisoformat('2022-05-30')
+        test_perf_alert_summary.triage_due = None
+
+        assert not test_perf_alert_summary.triage_due
+
+        test_perf_alert_summary.time_to_triage()
```

Comment:
This call doesn't do anything here, only returns the due date right? The `save() `is actually applying the change to `triage_due`. Therefore there's no need to call `time_to_triage()`

Reply:
not sure it is a good practice to just call `save()` as it is unintuitive. probably worth moving to `update_status()`